helm/fluent-logshipping-app/templates/configmap.yaml

```diff
@@ -48,10 +48,11 @@ data:
     {{- if has "syslog" .Values.outputs.inputLogTypes }}
     @INCLUDE inputs-syslog.conf
     {{- end }}
-    {{- if or (has "audit" .Values.outputs.inputLogTypes) (has "sshd" .Values.outputs.inputLogTypes) }}
+    {{- if or (has "audit" .Values.outputs.inputLogTypes) (has "sshd" .Values.outputs.inputLogTypes) (has "execve" .Values.outputs.inputLogTypes) }}
     @INCLUDE inputs-auditing.conf
     {{- end }}
 
+  {{- if has "containers" .Values.outputs.inputLogTypes }}
```

Comment:
This is not related to `execve`, right?

Reply:
No it is not, sorry, this is indeed a refactoring to prevent rendering this part as it is unused and caused me confusion when debugging